src/commands/init.js

```diff
@@ -71,28 +71,28 @@ module.exports = new Command("init [feature]")
 
     var choices = [
       {
-        name: "database",
-        label: "Database: Deploy Firebase Realtime Database Rules",
+        value: "database",
+        name: "Database: Deploy Firebase Realtime Database Rules",
```

Comment:
lol is that all that function did? Swap fields and rename?

Reply:
Yes.